storage/database/dynamodb.go

```diff
@@ -171,12 +182,14 @@ func NewDynamoDB(config *DynamoDBConfig) (*dynamoDB, error) {
 		switch tableStatus {
 		case dynamodb.TableStatusActive:
 			dynamoDB.logger.Warn("Successfully created dynamoDB table. You will be CHARGED until the DB is deleted.", "endPoint", config.Endpoint)
```

Comment:
How about moving this log in the `ReadOnly` if case?

Reply:
Both read only and non-read only is charged. I just add a log when it is read only.